src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -96,32 +98,45 @@ function onClick(
   }
 }
 
-export const DEFAULT_DARK_LOGO = 'opensearch_logo_dark_mode.svg';
-export const DEFAULT_LOGO = 'opensearch_logo_default_mode.svg';
+export const DEFAULT_LOGO = 'plugins/home/assets/logos/opensearch_dashboards.svg';
+export const DEFAULT_DARK_LOGO = 'plugins/home/assets/logos/opensearch_dashboards_darkmode.svg';
+
 interface Props {
   href: string;
   navLinks$: Observable<ChromeNavLink[]>;
   forceNavigation$: Observable<boolean>;
   navigateToApp: (appId: string) => void;
   branding: ChromeBranding;
+  basePath: HttpStart['basePath'];
+  // What background is the logo appearing on; this is independent of theme:darkMode
+  theme?: EuiHeaderProps['theme'];
 }
 
-export function HeaderLogo({ href, navigateToApp, branding, ...observables }: Props) {
+export function HeaderLogo({
+  href,
+  navigateToApp,
+  branding,
+  basePath,
+  theme = 'default',
+  ...observables
+}: Props) {
   const forceNavigation = useObservable(observables.forceNavigation$, false);
   const navLinks = useObservable(observables.navLinks$, []);
   const {
-    darkMode,
```

Comment:
Is there any reason that the `branding` service should even be wrapping `darkMode`? Should we have an additional issue or PR to remove that prop?

Reply:
I don't think we should remove it. `branding.darkMode` is the generally the source of truth and so far, we only have the `Header` which has its own `theme` and overrides the `branding.darkMode`.